### PR TITLE
[4.0] FIX SQL int default value to  numeric

### PR DESCRIPTION
**Pull request by amitranjan2**
Pull Request for Issue #23668

### Summary of Changes

INT DEFAULT value changed to numeric from char 

### Steps to reproduce the issue
joomla-cms/administrator/components/com_finder/sql/install.mysql.sql

Line 77 

 `hits` INT(11) NOT NULL DEFAULT '1', 

### Expected result
same as before

### Actual result

same as before 

### Documentation Changes Required
NO

### CODE REVIEW

File: build/media/com_menus/css/admin-item-edit_modules.css [hunk 1]
```diff
@@ -15,3 +15,4 @@ ul.horizontal-buttons li {
 ul.horizontal-buttons li .control-group {
 	margin-bottom: 0;
 }
+
```
Comment (alikon) [inline build/media/com_menus/css/admin-item-edit_modules.css]: why this file in the PR ?

**Review by alikon — CHANGES_REQUESTED** [review]

File: build/media/com_menus/css/admin-item-edit_modules.css (hunk 1 above)
Comment (amitranjan2) [reply build/media/com_menus/css/admin-item-edit_modules.css]: I removed

**Comment by alikon** [comment]
don't forget to take care of postgresql too
for example https://github.com/joomla/joomla-cms/blob/4.0-dev/installation/sql/postgresql/joomla.sql#L757

**Comment by amitranjan2** [comment]
Done

**Comment by amitranjan2** [comment]
@alikon  I think it's fine now, please review this

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on d69fbaec6f419c6b471677ad7843107753c13c0c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23669">issues.joomla.org/tracker/joomla-cms/23669</a>.</sub>

**Comment by anuragteapot** [comment]
I have tested this item :white_check_mark: successfully on 15d77e6f007d0745743aeda0f7331c82b7e3e19b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23669">issues.joomla.org/tracker/joomla-cms/23669</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23669">issues.joomla.org/tracker/joomla-cms/23669</a>.</sub>

**Comment by wilsonge** [comment]
Thankyou very much for your first contribution! Much appreciated!

**Comment by amitranjan2** [comment]
Thanks a lot

**Pull request merged**